nixos/modules/services/monitoring/prometheus/exporters.nix

```diff
@@ -301,6 +302,21 @@ in
         Please specify either 'services.prometheus.exporters.sql.configuration' or
           'services.prometheus.exporters.sql.configFile'
       '';
+    } {
+      assertion = cfg.scaphandre.enable -> (pkgs.stdenv.hostPlatform.isx86_64 == true);
+      message = ''
+        Only x86_64 host platform architecture is not supported.
+      '';
+    } {
+      assertion = cfg.scaphandre.enable -> ((lib.kernel.whenHelpers pkgs.linux.version).whenOlder "5.11" true).condition == false;
+      message = ''
+        A kernel version newer than '5.11' is required. ${pkgs.linux.version}
```

Comment:
This should explain that the assertion comes from `cfg.scaphandre.enable`.

Reply:
Will do